modules/github-action/commands/build.ts

```diff
@@ -16,27 +16,19 @@ export const builder: Builder<Args> = (yargs) =>
 		builders.withWorkspaces(
 			yargs
 				.usage('$0 build [options]')
-				.version(false)
 				.example('$0 build', 'Build all workspaces.')
 				.example('$0 build -w graph', 'Build the `graph` workspace only.')
 				.example('$0 build -w graph cli logger', 'Build the `graph`, `cli`, and `logger` workspaces.'),
 		),
 	);
 
 export const handler: Handler<Args> = async function handler(argv, { graph }) {
-	const workspace = graph.getByLocation(__dirname);
-
-	const [esbuildBin] = await run({
-		name: 'Get esbuild bin',
-		cmd: 'yarn',
-		args: ['bin', 'esbuild'],
-		opts: { cwd: graph.root.location },
-	});
+	const workspace = graph.getByName('github-action');
 
 	const files = await glob('*.ts', { cwd: workspace.resolve('src') });
 	const procs = files.map((file) => ({
 		name: `Build ${file}`,
-		cmd: esbuildBin,
+		cmd: 'esbuild',
```

Comment:
what changed to make it not necessary to get the full path to the esbuild binary?